public/index.html

```diff
@@ -5,8 +5,7 @@
     <meta name="viewport" content="width=device-width,initial-scale=1,minimal-ui" />
     <link rel="shortcut icon" type="image/x-icon" href="https://decentraland.org/images/icons/favicon-32x32.png" />
     <link rel="preload" href="%PUBLIC_URL%/images/background-v3@blur.jpg" as="image" />
-    <link rel="preload" href="%PUBLIC_URL%/images/background-v3@2x.jpg" as="image" />
-    <link rel="preload" href="%PUBLIC_URL%/images/loading.gif" as="image" />
+    <link rel="preconnect" href="https://api.thegraph.com" />
```

Comment:
I don't know when/if the explorer will start using the subgraph worker instead of api.thegraph directly so we might need to remember to change this in the future

Reply:
don't worry, we are doing follow up @fzavalia